### DISCUSSION TITLE
SunbirdRC Filters

### DISCUSSION THREAD

**Discussion by ManojNathIC** [Sunbird-RC/community#215] [General]
Currently sunbirdRC has only eq and startWith filters present, We need more filters min and max date (range) filter and number filter. 
@coolbung @SaketSinha

**Comment by tejash-jl** [comment]
@ManojNathIC these are the filter operations supported by SunbirdRC,
https://github.com/Sunbird-RC/sunbird-rc-core/blob/main/java/registry/src/main/java/dev/sunbirdrc/registry/dao/SearchDaoImpl.java#L65

I think you can use the `between` operation.
